modules/Makefile.circleci

```diff
@@ -45,6 +45,8 @@ circle\:tag-latest:
 	@$(SELF) docker:login
 	@echo "INFO: Tagging latest"
 	@$(SELF) DOCKER_TAG=latest docker:push
+	@git tag --force "$(CIRCLE_BRANCH)-docker-latest"
+	@git push origin --force "tags/$(CIRCLE_BRANCH)-docker-latest"
```

Comment:
There's `circle:tag` which is responsible for tagging the `BUILD` and `COMMIT`. This is called everytime we do `circle:deploy-kubernetes`. It's important to note that `latest` is not tagged. This is because it was decided that deployments of non-`master` branches should not be `latest`.

Then there is `circle:tag-latest`. This is called explicitly when we want the docker build to receive the `latest` tag. It's called explicitly in the `.deployment.master.commands` block in `circle.yml`.

Here's an example:  (compare `master` with `staging`)
![image](https://cloud.githubusercontent.com/assets/52489/16813940/9ea11c44-48e8-11e6-83b0-d4ff3952345a.png)